hatch/src/cli/commands/build.rs

```diff
@@ -17,57 +16,65 @@ impl<'build> Build {
       name: "build"
     }
   }
+
+  pub fn execute(&self, project: &Project) -> HatchResult<()> {
+    task::generate_assets(&project).with_context(|e| {
+      format!("Failed to generate assets : {}", e)
+    })?;
+
+    if let Some(path) = project.path().to_str() {
+      let command = format!("cd {} && tup", path);
+      match task::platform_type() {
+        PlatformKind::Windows => {
+          let mut child =
+            process::Command::new("cmd")
+              .arg("/C")
+              .arg(command)
+              .spawn().with_context(|e| {
+              format!("failed to build project at `{}` : {}", &path, e)
+            })?;
+          child.wait()?;
+        },
+        _ => {
+          let mut child =
+            process::Command::new("sh")
+              .arg("-c")
+              .arg(command)
+              .spawn().with_context(|e| {
+              format!("failed to build project at `{}` : {}", &path, e)
```

Comment:
Is this error message accurate? It seems that the error here is a failure to run the project.

Can we fix the context of this error message?

Reply:
We are not running a project, this is building a project.

Reply:
Oh right.  In that case can we make the error message specific to failure to call tup?

Reply:
All tup output is sent directly to stdout; therefore, any errors with tup will be visible to the user.

Reply:
Then this will fail only when the tup executable cannot be found

Reply:
I believe it will fail if the command returns an error status code, which tup will return if it fails.

Reply:
Failed to compile?

Reply:
That is an assumption.  We are saying all we know to be true, that it failed to build.  The tup output will also be present and they will have the exact context for how it failed to build.

Reply:
Ok